4.Clients/WebApi/ClientApp/src/app/login/csoft-signin.component.html

```diff
@@ -1,29 +1,30 @@
 <form nz-form [formGroup]="loginForm" class="login-form" (ngSubmit)="submitForm()">
-        <nz-form-item>
-          <nz-form-control>
-            <nz-input-group>
-              <input type="text" nz-input formControlName="userName" placeholder="Username" />
-            </nz-input-group>
-            <nz-form-explain *ngIf="loginForm.get('userName')?.dirty && loginForm.get('userName')?.errors"
-              >Please input your username!</nz-form-explain
-            >
-          </nz-form-control>
-        </nz-form-item>
-        <nz-form-item>
-          <nz-form-control>
-            <nz-input-group [nzSuffix]="prefixLock">
-              <input type="password" nz-input formControlName="password" placeholder="Password" />
-            </nz-input-group>
-            <nz-form-explain *ngIf="loginForm.get('password')?.dirty && loginForm.get('password')?.errors"
-              >Please input your Password!</nz-form-explain
-            >
-          </nz-form-control>
-        </nz-form-item>
-        <nz-form-item class="login-button">
-          <nz-form-control>
-            <button nz-button class="login-form-button" [nzType]="'primary'">LOG IN</button>
-          </nz-form-control>
-        </nz-form-item>
-      </form>
-      <ng-template #prefixUser><i nz-icon type="user"></i></ng-template>
-      <ng-template #prefixLock><i nz-icon type="lock"></i></ng-template>
\ No newline at end of file
+  <nz-form-item>
+    <nz-form-control>
+      <nz-input-group>
+        <input type="text" nz-input formControlName="userName" placeholder="Username" />
+      </nz-input-group>
+      <nz-form-explain *ngIf="loginForm.get('userName')?.dirty && loginForm.get('userName')?.errors"
+        >Please input your username!</nz-form-explain
+      >
+    </nz-form-control>
+  </nz-form-item>
+  <nz-form-item>
+    <nz-form-control>
+      <nz-input-group [nzSuffix]="prefixLock">
+        <input type="password" nz-input formControlName="password" placeholder="Password" />
+      </nz-input-group>
+      <nz-form-explain *ngIf="loginForm.get('password')?.dirty && loginForm.get('password')?.errors"
+        >Please input your Password!</nz-form-explain
+      >
+    </nz-form-control>
+  </nz-form-item>
+  <nz-form-item class="login-button">
+    <nz-form-control>
+      <button nz-button class="login-form-button" [nzType]="'primary'">LOG IN</button>
```

Comment:
Pequeño cambio, quitar las [] del nzType. Como no se va a modificar no es necesario
```suggestion
      <button nz-button class="login-form-button" nzType="primary">LOG IN</button>
```